clients/src/main/java/org/apache/kafka/common/config/SaslConfigs.java

```diff
@@ -16,11 +16,25 @@
 import java.util.Collections;
 import java.util.List;
 
+
 public class SaslConfigs {
     /*
      * NOTE: DO NOT CHANGE EITHER CONFIG NAMES AS THESE ARE PART OF THE PUBLIC API AND CHANGE WILL BREAK USER CODE.
      */
 
+    public static final String SASL_MECHANISMS = "sasl.mechanisms";
+    public static final String SASL_MECHANISMS_DOC = "The list of enabled SASL mechanisms. The list may contain any mechanism for which a security provider is available. GSSAPI is the default mechanism.";
+    public static final String GSSAPI_MECHANISM = "GSSAPI";
+    public static final List<String> DEFAULT_SASL_MECHANISMS = Collections.singletonList(GSSAPI_MECHANISM);
+
+    public static final String SASL_CALLBACK_HANDLER_CLASS = "sasl.callback.handler.class";
```

Comment:
@rajinisivaram SASL_CALLBACK_HANDLER_CLASS must be part of ```addClientSaslSupport``` in order to take effect